packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -0,0 +1,51 @@
+:host a {
+  cursor: pointer;
+  color: var(--gcds-link-default);
+
+  &:not([aria-disabled='true']) {
+    text-decoration: underline;
+    text-underline-offset: var(--gcds-link-underline-offset);
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-link-decoration-thickness);
+  }
+
+  &:hover {
+    color: var(--gcds-link-hover);
+    text-decoration-thickness: var(--gcds-link-hover-decoration-thickness);
+  }
+
+  &:focus {
+    border-radius: var(--gcds-link-focus-border-radius);
+    background-color: var(--gcds-link-focus-background);
+    color: var(--gcds-link-focus-text);
+    box-shadow: var(--gcds-link-focus-box-shadow);
+    outline: var(--gcds-link-focus-outline-width) solid var(--gcds-link-focus-background);
+    outline-offset: var(--gcds-link-focus-outline-offset);
+    border-color: var(--gcds-link-focus-border-color);
```

Comment:
Apologies for not catching this when you created the token but since the border colour + background colour are supposed to be the same colour, you can use the `--gcds-link-focus-background` token here instead of creating a separate token for the focus border. Can you please remove that token from the token repo? Again sorry for not catching this in the token PR!